### PR TITLE
Pos 972/1 - Enable nested virtualization

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

**Comment by aevesdocker** [comment]
Preview available here:
https://deploy-preview-15104--docsdocker.netlify.app/desktop/nested-virtualization/

Once published, the link will be 

https://docs.docker.com/desktop/nested-virtualization/

File: desktop/nested-virtualization.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note: 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
+{: .important}
+
+
+In general, Docker recommends running Docker Desktop natively on either a Windows system, to work with Windows or Linux containers, or on Mac or Linux, to work with Linux containers. However, Docker Desktop can run inside a virtual machine provided the virtual machine is properly configured.
```
[the remark this answers is not in the document]
Comment (christophermclellan) [reply desktop/nested-virtualization.md:12]: @aevesdocker - once we officially announce VDI support at the end of July we should then remove the line about recommending Mac or Windows.

File: desktop/nested-virtualization.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+description: Instructions on how to enable nested virtualization
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note: 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
```
[the remark this answers is not in the document]
Comment (MihaelaStoica) [reply desktop/nested-virtualization.md:8]: Yes, just Citrix.

File: desktop/nested-virtualization.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (aevesdocker) [reply desktop/nested-virtualization.md:12]: Made a note of this 👍

File: desktop/nested-virtualization.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+{: .important}
+
+
+In general, Docker recommends running Docker Desktop natively on either a Windows system, to work with Windows or Linux containers, or on Mac or Linux, to work with Linux containers. However, Docker Desktop can run inside a virtual machine provided the virtual machine is properly configured. 
+
+To run Docker Desktop inside a virtual machine, it is essential nested virtualization is enabled on the virtual machine (host VM). This is because, under the hood, Docker Desktop is using a Linux VM in which it runs Docker engine and the containers.
```
[the remark this answers is not in the document]
Comment (MihaelaStoica) [reply desktop/nested-virtualization.md:14]: Nested virtualization has to be enabled on the VM, not the host hypervisor, so `host VM` (or simply 'VM') would be the correct term. Customers may be using Virtual Machines directly, or Virtual Desktops (similar terms, but not quite the same). The instructions linked in this article (from Microsoft, VMWare, etc.) are generally referring to VMs, but if we want to use 'Virtual Desktop Environment', we could say:
`To run Docker Desktop in a Virtual Machine or in a Virtual Desktop Environment, it is essential that nested virtualization is enabled on the virtual machine or the virtual machine that provides the virtual desktop.`
or, even simpler: 
`To run Docker Desktop in a Virtual Desktop Environment, it is essential that nested virtualization is enabled on the virtual machine that provides the virtual desktop.`

File: desktop/nested-virtualization.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MihaelaStoica) [reply desktop/nested-virtualization.md:8]: Yes, we haven't validated Citrix yet

File: desktop/nested-virtualization.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+keywords: nested virtualization, Docker Desktop
+title: Enable nested virtualization
+---
+>Note: 
+>
+>This is work in progress. The steps haven’t been validated yet in our test environment.
+{: .important}
+
+
+In general, Docker recommends running Docker Desktop natively on either Mac, Linux, or Windows. However, Docker Desktop can run inside a virtual machine provided the virtual desktop is properly configured.
```
Comment (MihaelaStoica) [inline desktop/nested-virtualization.md:12]: here, we either say virtual machine or virtual desktop on both instances:
_However, Docker Desktop can run inside a **virtual machine** provided the **virtual desktop** is properly configured._

**Review by christophermclellan — APPROVED** [review]

**Pull request merged**